python/adelphi/adelphi/store.py

```diff
@@ -96,3 +101,28 @@ def set_replication_factor(selected_keyspaces, factor):
             log.debug("Replication for keyspace " + ks.name+ ": " + str(ks.replication_strategy))
             strategy = ks.replication_strategy
             strategy.replication_factor_info = factor
+
+
+def create_schema(session, schemaPath):
```

Comment:
Can we move this to the test module?

Reply:
We probably could, but I'd argue this function belongs in adelphi.store just as much as with_cluster() or with_local_cluster().  It seems like the bigger problem here is that it's poorly named.  This function exists to execute a collection of semicolon-terminated CQL statements contained in a file against a CQL instance somewhere.  We use it in the tests for setting up schemas but you can imagine any number of usages other than that within Adelphi.

I think the better answer would be to rename it to execute_cql() or something like that.  wdyt?